example/custom-singularity.config

```diff
@@ -40,7 +40,7 @@ params {
 }
 
 process {
-  container = "${params.containerPath}/rnafusion_1.1.0.img"
+  container = "${params.containerPath}/rnafusion_${container_version}.img"
```

Comment:
I think this should be: rnafusion_v${container_version}.img
Taken from here: https://github.com/nf-core/rnafusion/blob/505a0fb784771fdb5d2e9c3e28fccb490430727d/download-singularity-img.nf#L102